package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "xml",
   "version": "1.2.1",
```

Comment:
You forgot to bump minor version of the component. This update should be described in short form in the `CHANGELOG.md` as well.